docs/getting-started/offline-endpoint.asciidoc

```diff
@@ -97,6 +99,8 @@ Complete these steps:
 
 Deploy an HTTP file server to serve files from a local directory, which will be filled with artifact update files in a later step.
 
+IMPORTANT: The entity tag (Etag) header is a mandatory HTTP response header that you _must_ set in your server configuration file. {elastic-endpoint} uses the Etag header to determine whether your global artifacts have been updated since they were last downloaded. If your server configuration file does not contain an ETag header, {elastic-endpoint} won't download new artifacts when they're available.
```

Comment:
```suggestion
IMPORTANT: The entity tag (`Etag`) header is a mandatory HTTP response header that you _must_ set in your server configuration file. {elastic-endpoint} uses the `Etag` header to determine whether your global artifacts have been updated since they were last downloaded. If your server configuration file does not contain an `ETag` header, {elastic-endpoint} won't download new artifacts when they're available.
```